src/starknet_client/mod.rs

```diff
@@ -317,25 +329,24 @@ impl<P: Provider + Send + Sync> KakarotClient<P> {
         newest_block: BlockNumberOrTag,
         _reward_percentiles: Option<Vec<f64>>,
     ) -> Result<FeeHistory, EthApiError> {
-        let block_count_usize =
-            usize::try_from(block_count).map_err(|e| ConversionError::ValueOutOfRange(e.to_string()))?;
+        let block_count_usize = usize::try_from(block_count + U256::from(1))
```

Comment:
any reason to add 1?

Reply:
yes but I think this causes an issue lower in the file when we do the `gas_used_ratio` lower in the function

Reply:
ok will check